src/simplewallet/simplewallet.cpp

```diff
@@ -3257,6 +3284,7 @@ bool simple_wallet::set_variable(const std::vector<std::string> &args)
     success_msg_writer() << "track-uses = " << m_wallet->track_uses();
     success_msg_writer() << "setup-background-mining = " << setup_background_mining_string + tr(" (set this to support the network and to get a chance to receive new monero)");
     success_msg_writer() << "device_name = " << m_wallet->device_name();
+    success_msg_writer() << "export-format = " << (m_wallet->export_format() ? "Hex" : "Raw");
```

Comment:
Can this be "m_wallet->export_format() == Hex ? "Hex" : "Raw"" ? Doesn't depend on enum values this way.

Reply:
Good point. Fixing.